apps/champions/lib/champions/items.ex

```diff
@@ -48,5 +48,5 @@ defmodule Champions.Items do
   end
 
   defp calculate_level_up_cost(item),
-    do: {Currencies.get_currency_by_name!("Gold").id, item.level |> Math.pow(2) |> round()}
+    do: [{Currencies.get_currency_by_name!("Gold").id, item.level |> Math.pow(2) |> round()}]
```

Comment:
Why this change? Does it matter if the tuple is inside a list or not?

Reply:
It does because we want to use substract_currencies, which takes in a list! This function is just a placeholder until we get actual level up cost logic anyways, what matters is that it follows the data type we'll want to have when we do implement it